### PR TITLE
Error if describeApi.only is seen

### PR DESCRIPTION
**Pull request by matttrawicki**
[sc-387185]

### CODE REVIEW

**Review by dmg3 — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -69,8 +70,12 @@ module.exports = {
                 'import/export': 'off',
 
                 // no-only-tests
-                'no-only-tests/no-only-tests': 'error',
-
+                'no-only-tests/no-only-tests': [
+                    'error', {
+                        // describeApi is from our new route testing helper
+                        'block': ['describeApi'].concat(noOnlyTest.rules['no-only-tests'].meta.schema[0].properties.block.default)
```
Comment (odielleongc) [inline index.js:76]: you need it also for `scenario.only`

Comment (matttrawicki) [reply index.js:76]: That'll cost you extra 💰

Comment (odielleongc) [reply index.js]: all I have is an empty wallet

**Review by analisegiobbi3 — APPROVED** [review]
Thanks!! You rock!

**Pull request merged**